templates/dashboard/apps.html

```
@@ -0,0 +1,244 @@
+        <p>No apps connected yet. Install an app below to get started.</p>
+    </div>
+    {% endif %}
+</section>
+
+<!-- Available Apps -->
+{% if available %}
+<section class="apps-section">
+    <h2 class="section-title">Available</h2>
+    <div class="apps-grid">
+        {% for app in available %}
+        <a class="app-card app-card--link"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)